src/coreclr/jit/instr.cpp

```diff
@@ -721,7 +721,7 @@ void CodeGen::inst_TT_RV(instruction ins, emitAttr size, GenTree* tree, regNumbe
     unsigned varNum = tree->AsLclVarCommon()->GetLclNum();
     assert(varNum < compiler->lvaCount);
 #if CPU_LOAD_STORE_ARCH
-    assert(GetEmitter()->emitInsIsStore(ins));
+    assert(GetEmitter()->emitInsIsStore(ins) || (ins == INS_sve_str));
```

Comment:
Is this change related?

Reply:
It is somewhat related to https://github.com/dotnet/runtime/issues/105512, but I wanted to point fix it here and we can revert this when we properly fix https://github.com/dotnet/runtime/issues/105512.